### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by zcraber**
Added Doctor Live, Grace Family TV, Kalika TV, Starnet and CCV.

### CODE REVIEW

File: streams/in.m3u [hunk 1]
```diff
@@ -394,6 +412,8 @@ http://103.199.161.254/Content/manoramanews/Live/Channel(ManoramaNews)/index.m3u
 http://103.199.161.254/Content/mathrubhuminews/Live/Channel(Mathrubhuminews)/index.m3u8
 #EXTINF:-1 tvg-id="MathrubhumiNews.in",Mathrubhumi News (1080p) [Not 24/7]
 https://ythls.onrender.com/channel/UCwXrBBZnIh2ER4lal6WbAHw.m3u8
+#EXTINF:-1 tvg-id="Mayyazhi.in",Mayyazhi (720p)
```
Comment (PopeyeTheSai10r) [inline streams/in.m3u:415]: Either not 24/7 or Geo-blocker.

File: streams/in.m3u [hunk 2]
```diff
@@ -702,8 +726,14 @@ https://ythls.onrender.com/channel/UCDCMjD1XIAsCZsYHNMGVcog.m3u8
 https://6n3yope4d9ok-hls-live.5centscdn.com/vaanavil/TV.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="VanithaTV.in",Vanitha TV (720p) [Not 24/7]
 https://ythls.onrender.com/channel/UCTcH-GOkexQHWKBWSfOjhPQ.m3u8
+#EXTINF:-1 tvg-id="VBCNews.in",VBC News (576p)
+https://5a1178b42cc03.streamlock.net/8200/8200/playlist.m3u8
+#EXTINF:-1 tvg-id="VCV.in",VCV (720p)
+https://5a1178b42cc03.streamlock.net/8210/8210/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/in.m3u:732]: Not loading for me. Either Not 24/7 or Geo-Blocked.

File: streams/in.m3u [hunk 3]
```diff
@@ -820,9 +850,13 @@ http://150.242.21.81:8088/hls/ubl.m3u8
 https://932y483pdjv8-hls-live.5centscdn.com/stream/deb10bae362f810630ec3abedcae5894.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="",Vathanam TV
 http://95.216.167.183:5080/LiveApp/streams/443106610169904881506470.m3u8
+#EXTINF:-1 tvg-id="Vision.in",Vision (1080p)
+http://103.85.204.205:1935/VISIONMEDIA/live/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/in.m3u:854]: Same as the others. Not 24/7 or Geo.

File: streams/in.m3u [hunk 4]
```diff
@@ -820,9 +850,13 @@ http://150.242.21.81:8088/hls/ubl.m3u8
...
 #EXTINF:-1 tvg-id="",Vyas NIC (480p)
 #EXTVLCOPT:http-referrer=https://webcast.gov.in/
 https://hls.media.nic.in/hls/live/vyas/vyas.m3u8
+#EXTINF:-1 tvg-id="WeOneKerala.in",We One Kerala (720p)
+http://kl.logicwebs.in:1935/sreemedia/sreemedia/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/in.m3u:859]: Same as the others. Not 24/7 or Geo.

File: streams/in.m3u (hunk 2 above)
Comment (zcraber) [reply streams/in.m3u:732]: Accessible using VPN. So, not Geo-Blocked. 
image

File: streams/in.m3u (hunk 1 above)
Comment (zcraber) [reply streams/in.m3u:415]: Accessible from Europe (VPN). So, not Geo-Blocked. 

image

File: streams/in.m3u (hunk 3 above)
Comment (zcraber) [reply streams/in.m3u:854]: Accessible from Europe (VPN). So, not Geo-Blocked.

image

File: streams/in.m3u (hunk 4 above)
Comment (zcraber) [reply streams/in.m3u:859]: Not accessible from Europe (VPN). So, it's Geo-Blocked.

image

**Review by freearhey — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**